### PR TITLE
CORS repare for api, fix #3367

### LINKED ISSUE #3367: CORS doesn't work?
Maybe I am wrong, but it seems that CORS in wallabag doesn't work?
OPTIONS preflight request returns an error 400. It happends because of default settings of the **nelmioCors** bundle in config.yml: 
`allow_headers: ['X-Custom-Auth']`
but we need the `Authorization` header to be allowed. 

When I changed that setting to `allow_headers: ['Authorization']` CORS request begin working.

Does somebody need 'X-Custom-Auth' header?
Should I make a PR?

### PR DESCRIPTION
**Pull request by Rurik19**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| Fixed tickets | #3367

### CODE REVIEW

**Comment by j0k3r** [comment]
Why not updating others `allow_headers`?

**Comment by j0k3r** [comment]
Could you rebase your branche again recent changes from the 2.3 branch?

**Comment by Rurik19** [comment]
done

**Comment by nicosomb** [comment]
@Rurik19 Do you still work on this PR.

**Comment by Rurik19** [comment]
@nicosomb  for now, this last commit is enough,

File: app/config/config.yml [hunk 1]
```diff
@@ -157,11 +157,11 @@ nelmio_cors:
             max_age: 3600
         '^/':
             #origin_regex: true
-            allow_origin: ['^http://localhost:[0-9]+']
+            allow_origin: ['*']
             allow_headers: ['Authorization','content-type']
             allow_methods: ['POST', 'PUT', 'GET', 'DELETE']
             max_age: 3600
-            hosts: ['^api\.']
+            #hosts: ['^api\.']
```
Comment (j0k3r) [inline app/config/config.yml]: Why should we allow cors on the UI? 🤔

Comment (Rurik19) [reply app/config/config.yml]: As we still don't have API endpoints for clients information, I want to get application info right from the ui, like native applications do. If you thind it is unsecuse, let us kill last commit.

Comment (j0k3r) [reply app/config/config.yml]: By client info, you mean the credentials?
I don't really like that way to do. It'll work for you, but it'll also open other security issues.

Comment (Rurik19) [reply app/config/config.yml]: not password/username but clientId and clientSecret

Comment (Rurik19) [reply app/config/config.yml]: And then - CORS restriction works only for web apps, any local programs, even **curl** or **wget** can grab pages easily.

**Pull request closed without merging**